src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -695,6 +711,52 @@ private void determineUnitsToAttackWith(final List<ProTerritory> prioritizedTerr
       Map<Unit, Set<Territory>> sortedUnitAttackOptions =
           tryToAttackTerritories(prioritizedTerritories, alreadyMovedUnits);
 
+      // Clear bombers
```

Comment:
Data encapsulation violation, looks like it should be a `clearBombers()` method that does this functionality. A `attackMap` 1st class object looks like a way to accomplish that.